src/main/java/tornadofx/Builders.kt

```diff
@@ -109,6 +109,12 @@ fun Pane.progressbar(property: Property<Double>, op: (ProgressBar.() -> Unit)? =
     progressProperty().bind(property)
 }
 
+fun Pane.slider(min: Double? = null, max: Double? = null, value: Double? = null, op: (Slider.() -> Unit)? = null) = opcr(this, Slider().apply {
```

Comment:
I would not add the orientation, use the default orientation or adapt in the op block (my 0.2 cents)